src/components/transport_manager/test/transport_manager_impl_test.cc

```diff
@@ -633,15 +633,15 @@ TEST_F(TransportManagerImplTest, SendMessageToDevice_SendDone) {
   EXPECT_TRUE(waiter.WaitFor(1, kAsyncExpectationsTimeout));
 }
 
-TEST_F(TransportManagerImplTest, SendMessageFailed_GetHandleSendFailed) {
+TEST_F(TransportManagerImplTest, SendMessageToDevice_SendFailedEvent) {
   // Arrange
   HandleConnection();
 
   TestAsyncWaiter waiter;
   EXPECT_CALL(*mock_adapter_,
               SendData(mac_address_, application_id_, test_message_))
-      .WillOnce(DoAll(NotifyTestAsyncWaiter(&waiter),
-                      Return(TransportAdapter::FAIL)));
+      .WillOnce(
+          DoAll(NotifyTestAsyncWaiter(&waiter), Return(TransportAdapter::OK)));
```

Comment:
@mvorobio Unclear change. In FAILED test case you return `OK`. Looks really strange

Reply:
@AByzhynar Adapter's `SendData` returned value is only about a status of underlying layers as data transmission is asynchronous. The real transmission status is distributed via events `ON_SEND_DONE` or `ON_SEND_FAIL`.